thoth/common/openshift.py

```diff
@@ -77,6 +77,14 @@ def __init__(
                 "installed with openshift extras?"
             ) from exc
 
+        try:
+            import argo.workflows as argo
+        except ImportError as exc:
+            raise ImportError(
+                "Unable to import argo-workflows package. Was thoth-common library "
+                "installed with argo extras?"
+            ) from exc
+
```

Comment:
We currently do not maintain any extra on this package. I removed them - IMHO having argo as a separate extra does not make sense, we can have `argo+openshift` bundled in one extra if that would be desired (might simplify dependency management later).

Reply:
Ah I see, I somehow inferred that from the OpenShift/Kubernetes import check above. +1, we can add argo+openshift extras